src/main/java/seedu/moneygowhere/exceptions/ConsoleParserCommandDeleteIncomeInvalidException.java

```diff
@@ -0,0 +1,28 @@
+package seedu.moneygowhere.exceptions;
+
+@SuppressWarnings("unused")
+public class ConsoleParserCommandDeleteIncomeInvalidException extends MoneyGoWhereException {
```

Comment:
Is it possible to add a Javadoc to describe the circumstance in which this exception will be thrown?